### DISCUSSION TITLE
Add missing languages into the game

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21771] [Q&A] [answered]
There are 39 languages excluding Malay, Tajik, Sinhala (which are 0% translated) but game only has 29 of them.
Missing are Filipino, Catalan, Arabic, Lithuanian, Serbian, Hebrew, Croatian, Slovak, Persian and Latvian.
Same with web excluding Arabic.

**Comment by frenzibyte** [comment]
RTL languages such as Arabic, Hebrew, and Persian won't be added anytime soon as our text renderer does not have proper support for it yet (+ it may require changes to the direction of the UI to actually make sense visually)

**Reply by Loreos7** [reply]
Could you create tracking issue for that?

**Reply by frenzibyte** [reply]
https://github.com/ppy/osu-framework/issues/89

**Comment by peppy** [comment, marked as answer]
Languages will be added as we find time and get requests for them.
